packages/js/src/structured-data-blocks/how-to/components/HowToStep.js

```diff
@@ -336,7 +335,7 @@ export default class HowToStep extends Component {
 					placeholder={ __( "Enter a step title", "wordpress-seo" ) }
 					allowedFormats={ [ "core/italic", "core/strikethrough", "core/link", "core/annotation" ] }
 				/>
-				<RichTextWithAppendedSpace
+				<RichText
```

Comment:
For my learning purpose: Looking at this change and also the change in `HowTo.js`, we don't need to add appended space for `RichText` that are rendered in the editor and not in the front-end. Is this correct assumption? and if yes, what is the reasoning behind it?

Reply:
Yeah, I should have documented that change, sorry for that! I'm basically following how we do it for the FAQ -- just using `RichText` for the editor instead of a higher-order component. Appending a space in the editor is nonsensical as it doesn't accomplish anything 😅

Reply:
No worries! Thank you for the explanation! 😄